cookbooks/aws-parallelcluster-common/resources/network_service/partial/_restart_network_alinux_centos.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :restart do
+  network_service_name = 'network'
+  Chef::Log.info("Restarting '#{network_service_name}' service, platform #{node['platform']} '#{node['platform_version']}'")
+
+  service network_service_name.to_s do
```

Comment:
Nitpick: why `to_s`?

Reply:
This was already there in the original code. I'll remove it.

Reply:
Removed.